Black-Market/Black-Market/Protocols/UserNetworkInterface.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  UserNetworkInterface.swift
+//  Black-Market
+//
+//  Created by Junior Sancho on 8/2/23.
+//
+import Foundation
+import Combine
+
+protocol UserNetworkInterface {
+    func signUp(params: [String: Any]) -> AnyPublisher<DefaultResponse, Error>
+    func signIn(params: [String: Any]) -> AnyPublisher<UserDataResponse, Error>
```

Comment:
Maybe a good idea is to keep the dictionary structure of data at the networking and service layers only, and allow clients to call these functions with the data needed only(and required from the bizz logic), like email, password, etc

Reply:
Done